pkg/features/kube_features.go

```diff
@@ -373,6 +373,15 @@ const (
 	// Allow users to specify recreating pods of a job only when
 	// pods have fully terminated.
 	JobPodReplacementPolicy featuregate.Feature = "JobPodReplacementPolicy"
+
+	// owner: @tenzen-y
+	// kep : https://kep.k8s.io/3998
+	// alpha: v1.30
+	//
+	// Allow users to specify when a Job can be declared as succeeded
+	// based on the number of succeeded pods.
```

Comment:
It is not just based on the "number". Maybe just shorter is better in this case: "Allow users to specify when a Job can be declared as succeeded.", or if we want longer version then "Allow users to specify when a Job can be declared as succeeded based on the set of succeeded pods."

Reply:
SGTM. Thank you for this rephrasing.